src/components/rooms-tab.yml

```diff
@@ -0,0 +1,24 @@
+ownedRoomsHeader:
+  en: Your rooms
+  de: Deine Räume
+joinedRoomsHeader:
+  en: Rooms you are a member of
+  de: Räume in denen du Mitglied bist
+accessType_public:
+  en: public
+  de: öffentlich
+accessType_private:
+  en: private
+  de: privat
+name:
+  en: Name
+  de: Name
+access:
+  en: Access
+  de: Zugang
+joinedOn:
+  en: Joined on
+  de: Eingetreten am
+owner:
+  en: Owner
+  de: Besitzer
```

Comment:
@ahelmberger Would this be the proper translation for room owner?

Reply:
I used Eigentümer, if so, I'll also change my PR to have consistency

Reply:
Extracted to `owner` to `common`